File: http-server-netty/src/test/groovy/io/micronaut/http/server/netty/websocket/Message.java
package io.micronaut.http.server.netty.websocket;

import java.util.Objects;

public class Message {

    private String text;

    public Message(String text) {
        this.text = text;
    }

    Message() {
    }

    public String getText() {
        return text;
    }

    public void setText(String text) {
        this.text = text;
    }

    @Override
    public boolean equals(Object o) {
        if (this == o) return true;
        if (o == null || getClass() != o.getClass()) return false;
        Message message = (Message) o;
        return Objects.equals(text, message.text);
    }

    @Override
    public int hashCode() {
        return Objects.hash(text);
    }

    @Override
    public String toString() {
        return "Message{" +
                "text='" + text + '\'' +
                '}';
    }
}

File: test-suite-python/src/test/python/micronaut/docs/http/server/netty/websocket/Message.py
from dataclasses import dataclass

from micronaut.core.annotation import Introspected, ReflectiveAccess


@ReflectiveAccess
@Introspected
@dataclass
class Message:
    text: str | None = None

    def getText(self) -> str | None:
        return self.text

    def setText(self, text: str) -> None:
        self.text = text

    def __str__(self) -> str:
        return "Message{text='" + str(self.text) + "'}"
